### PR TITLE
Feature/Implement getTransaction for crypto transfer

### PR DESCRIPTION
**Pull request by natanasow**
**Description**:

**Related issue(s)**:

Fixes #
- https://github.com/hashgraph/hethers.js/issues/156

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/tests/src.ts/test-wallet.spec.ts [hunk 1]
```diff
@@ -706,4 +706,30 @@ describe("Wallet createAccount", function () {
 
         assert.strictEqual(exceptionThrown, true);
     }).timeout(timeout);
+
+    it("Should be able to get a crypto transfer transaction via provider.getTransaction(tx.transactionId)", async function () {
+        const transaction = await acc1Wallet.sendTransaction({
+            to: acc2Wallet.account,
+            value: 1.8925
```
[the remark this answers is not in the document]
Comment (natanasow) [reply packages/tests/src.ts/test-wallet.spec.ts:713]: hbar

File: packages/providers/src.ts/base-provider.ts [hunk 2]
```diff
@@ -552,6 +552,24 @@ export class BaseProvider extends Provider {
...
+                                charityFee = filtered[0].charged_tx_fee - t.amount;
+                                return false;
+                            }
+                            return t.account != record.from;
+                        }).filter(function (t: any) {
+                            return t.amount != charityFee;
+                        });
+
+                        record.to = toTransfers[0].account;
+                        record.amount = toTransfers.reduce((a: any, b: any) => a + b.amount, 0);
```
Comment (Daniel-K-Ivanov) [inline packages/providers/src.ts/base-provider.ts:572]: Use Custom data for multiple transfers

**Review by Daniel-K-Ivanov — CHANGES_REQUESTED** [review]

File: packages/abstract-provider/src.ts/index.ts [hunk 3]
```diff
@@ -50,7 +50,8 @@ export type HederaTransactionRecord = {
     block_number?: number,
     hash: string,
     logs?: {},
-    accountAddress?: string
+    accountAddress?: string,
+    transfersList?: Array<{ to: string, amount: number }>
```
Comment (joan41868) [inline packages/abstract-provider/src.ts/index.ts:54]: I think that this transfer list will work **only** for `Hbar` transfers.
Will we ever support HTS (hedera token service) tokens?

Comment (joan41868) [reply packages/abstract-provider/src.ts/index.ts:54]: We won't support `HTS` tokens. You can resolve this.

**Review by Daniel-K-Ivanov — APPROVED** [review]

**Pull request merged**